### PR TITLE
Add the current state to tenant details. Fix Enable/Disable Encryption

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2021-05-17 at 9 57 13 AM

Screen Shot 2021-05-17 at 10 57 51 AM

Screen Shot 2021-05-17 at 10 57 58 AM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/TenantDetails/TenantDetails.tsx [hunk 1]
```diff
@@ -510,6 +510,10 @@ const TenantDetails = ({
                           </React.Fragment>
                         )}
                       </tr>
+                      <tr>
+                        <td>State:</td>
+                        <td colSpan={3}>{tenant?.currentState}</td>
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/TenantDetails.tsx:515]: Not sure if this type of ternary works well in ES6, I couldn't find a reference for it

**Review by bexsoft — CHANGES_REQUESTED** [review]
Trying to run console I get the following error:

`TypeScript error in /Users/benjaminperez/dev/console/portal-ui/src/screens/Console/Buckets/ViewBucket/ViewBucket.tsx(855,48):
Cannot find name 'handleEncryptionCheckbox'.  TS2304`

File: portal-ui/src/screens/Console/Tenants/TenantDetails/TenantDetails.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/TenantDetails/TenantDetails.tsx:515]: we use it all over the place, it's unwrapping an optional value `ITenant | null`, if it's null, we won't access `currentState` avoiding NPE

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by Alevsk — APPROVED** [review]

**Pull request merged**